### ISSUE TITLE
How to sanitize using v4 API?

### ISSUE THREAD

**Issue by ijohn** [ctavan/express-validator#397]
Do we have to use the legacy API to sanitize?

**Comment by gustavohenke** [comment]
No, all you need is validator itself.

```js
const { stripLow, normalizeEmail } = require('validator');
const email = normalizeEmail(stripLow(req.body.email));
```

You can make this even cooler if you do some function composition using eg [`_.flow()`](https://lodash.com/docs/4.17.4#flow).

_For now_, I have no plans of adding further sanitization support to the new v4 APIs.
But I may change my mind, of course 😉

**Comment by gustavohenke** [comment]
Hi!
Sanitization is available starting with v4.2.0  :)

Check the docs for details on how to use it!
